python/lsst/ts/schedulerConfig/instrument/optics_loop_corr.py

```diff
@@ -34,6 +36,15 @@ def set_array(self, conf, param):
             The name of the topic parameter to fill.
         """
         array = getattr(conf, param)
-        local_param = getattr(self, param)
+        local_param = getattr(self, self.camelcase_to_snakecase(param))
         for i, v in enumerate(local_param):
             array[i] = v
+
+    def camelcase_to_snakecase(self, name):
```

Comment:
This seems more like a utility method than a class method. Maybe move this to something inside `python/lsst/sims/schedulerConfig/utilities` or make it a `@staticmethod`? I think it is general enough to be placed inside utilities...

Reply:
I did consider placing this method in the utilities class however this is the only place this method is used. 

Regardless agree with you this method doesn't belong here. However I don't feel moving into a utilities class will foster good software development.

I feel this method should be removed and either update the class attributes to be lowerCamelCase since by design the method `set_array()` requires both its own class attributes and the topic to be identical, therefore violating encapsulation. I did try to update the class methods but was getting more errors. Rather than chasing the rabbit hole further I added `camelcase_to_snakecase` as a quick fix.

Or update the `set_array()` to not directly reference attributes from another object but use an alternative mechanism for retrieving the data it wants.

Reply:
ok.. In this case, can you make a `@staticmethod`?